README.md

```diff
@@ -1,45 +1,38 @@
 
 # Home Office Posters
-Home Office repository of posters covering different topics - research, access needs, accessibility, design.
+Home Office repository of posters covering different topics - research, access needs, accessibility and design.  
 
+## Contributions and contact
 
+If you want to contribute a translation of one of our posters, we would really appreciate your help.
 
-**See ALL the posters**
+You are free to adapt the posters under [the licence](https://creativecommons.org/licenses/by-nc-sa/4.0/). But please remove the Home Office name and link back to this repo giving appropriate credit.
```

Comment:
The last section could be a little clearer (I was initially confused by it saying remove name but add link) - could we seperate out the sentence into bullets, for eg:

You are free to adapt the posters under the following conditions:
- the posters are available under the [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International licence](https://creativecommons.org/licenses/by-nc-sa/4.0/)
- remove the Home Office name and logo
- give appropriate credit and link back to this repo

Reply:
Thanks Chris - this looks great.

Reply:
Have adjusted so its clearer like suggested